shared/settings.lua

```diff
@@ -16,5 +23,5 @@ return {
     },
 
     -- Nearby object distance check.
-    nearbyObjectDistance = 20.0,
+    nearbyObjectDistance = 5.0,
```

Comment:
This needs to stay at 15.0-20.0 for drawing nearby pins with longer ranges of display.